src/track.jsx

```diff
@@ -68,7 +67,7 @@ var renderSlides = (spec) => {
     }
 
     var childStyle = getSlideStyle(assign({}, spec, {index: index}));
-    slides.push(cloneWithProps(child, {
```

Comment:
The warning I saw had advice to do it with `slides.push(React.cloneElement(child, {`, which removed the warning.